src/main/resources/db/migration/0350/V397__TFContactsUpdate.sql

```diff
@@ -0,0 +1,17 @@
+DROP INDEX IF EXISTS organization_users_contact_uk;
+
+CREATE TABLE project_user_roles
+(
+    id   INTEGER PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE project_internal_users
+(
+    project_id BIGINT NOT NULL,
+    user_id    BIGINT NOT NULL,
+    project_user_role_id    BIGINT REFERENCES project_user_roles,
+    role_name  TEXT,
+    PRIMARY KEY (project_id, user_id, project_user_role_id),
+    CONSTRAINT project_internal_users_role_exclusive CHECK ((project_user_role_id IS NULL) != (role_name IS NULL))
+);
```

Comment:
Do we need to seed this table with the existing TF Contact users?

Reply:
According to product:
> No, existing TF Contacts should not automatically become PLs or any other leads under Project Profiles. They can be manually associated with any Lead types in the Internal Contacts field.